content/docs/gke/deploy/deploy-cli.md

```diff
@@ -74,12 +74,14 @@ Follow these steps to deploy Kubeflow:
     export ZONE=<your target zone> #where the deployment will be created
 
     export PROJECT=<your GCP project>
+    # label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'kf',  or 'kfc-test'
```

Comment:
My review comment has disappeared into the depths of the file, so I'm adding it again here. :)

Thanks for adding this comment! The word "label" isn't clear, as it doesn't specify which label we mean. Instead, it's best to be specific - something like this:

```
# The value of KFAPP must consist of...
```

Another tech writer's best practice: It's best not to use "e.g." as many people don't know the difference between "e.g." and "i.e.". Instead, spell out the full term. Also, there's an unmatched parenthesis in the comment. Suggestion:

```
... character. For example, 'kf' or 'kfc-test'.
```